PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,13 @@
-Thank you for contributing to Servo! Please replace each `[ ]` by `[X]` when the step is complete, and replace `__` with appropriate data:
+Replace this paragraph with a description of the changes, then follow the instructions in the next section.
```

Comment:
Maybe this should also be a comment that just says "Enter a description of your changes here below this line:"

(I think that makes it a little harder to gloss over this step.)